test/unit/abci/handlers/beginBlockHandlerFactory.spec.js

```diff
@@ -22,6 +23,8 @@ describe('beginBlockHandlerFactory', () => {
   beforeEach(function beforeEach() {
     blockchainState = new BlockchainState();
 
+    protocolVersion = 1;
```

Comment:
Should be an instance of the Long class?

Reply:
Doing that right now.